tests/unittests/Foundation/NSAttributedStringTests.mm

```diff
@@ -547,8 +554,10 @@ void assertAttributeAt(
                      options:NSAttributedStringEnumerationReverse
                   usingBlock:^void(NSDictionary* val, NSRange range, BOOL* stop) {
                       ++timesEnumerated;
+                      ASSERT_EQ(0, [val count]);
+                      ASSERT_TRUE(NSEqualRanges(NSMakeRange(1, 3), range));
                   }];
-    ASSERT_EQ(0, timesEnumerated);

```

Comment:
Our implementation of enumerateAttribute was incorrect - it did not enumerate over ranges where there were no attributes. That was corrected in this CR.

---
In reply to: [73620275](https://github.com/Microsoft/WinObjC/pull/819#discussion_r73620275) [](ancestors = 73620275)